seumich/views.py

```diff
@@ -35,6 +35,7 @@ class CohortsListView(LoginRequiredMixin, UserLogPageViewMixin, ListView):
     queryset = Cohort.objects.filter(id__gte=0)
     context_object_name = 'cohorts'
 
+
```

Comment:
remove the line so that there is no need to include the file in PR

Reply:
Done. See [https://github.com/tl-its-umich-edu/student_explorer/pull/164/commits/b6438fc7affc49a1f1d2294066feaff7f61def84](https://github.com/tl-its-umich-edu/student_explorer/pull/164/commits/b6438fc7affc49a1f1d2294066feaff7f61def84)